util/chplenv/chpl_llvm.py

```diff
@@ -435,24 +438,35 @@ def get_system_llvm_clang(lang):
     # or, if CHPL_TARGET_COMPILER=llvm, CHPL_TARGET_CC/CXX
     tgt_llvm = overrides.get('CHPL_TARGET_COMPILER', 'llvm') == 'llvm'
     if lang_upper == 'C':
-        llvm_clang_c = overrides.get('CHPL_LLVM_CLANG_C', '')
-        if llvm_clang_c != '':
-            return llvm_clang_c
+        llvm_clang_c = overrides.get('CHPL_LLVM_CLANG_C')
+        if llvm_clang_c:
+            return llvm_clang_c.split()
```

Comment:
You may want to add a comment (maybe after line 438) explaining why `split()` is necessary (because you want to separate the command out from any arguments passed to it).